Mathlib/Topology/Algebra/Module/PointwiseConvergence.lean

```diff
@@ -0,0 +1,155 @@
+/-
+Copyright (c) 2024 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import Mathlib.Topology.Algebra.Module.StrongTopology
+import Mathlib.Topology.Algebra.Module.WeakDual
+
+/-!
+# Topology of pointwise convergence on continous linear maps
+
+## Main definitions
+
+* `PointwiseConvergenceCLM`: Type synonym of `E →SL[σ] F` equipped with the uniform convergence
+topology on finite sets.
+
+## Main statements
+
+* `fooBar_unique`
+
+## Notation
+
+
+
+## Implementation details
+
+
+
+## References
+
+* [F. Bar, *Quuxes*][bibkey]
+
+## Tags
+
+Foobars, barfoos
+-/
+
+/-! ### Topology of pointwise convergence -/
+
+variable {α ι : Type*} [TopologicalSpace α]
+variable {𝕜 𝕜₁ 𝕜₂ : Type*} [NormedField 𝕜] [NormedField 𝕜₁] [NormedField 𝕜₂]
+variable {σ : 𝕜₁ →+* 𝕜₂}
+variable {E F : Type*} [AddCommGroup E] [TopologicalSpace E]
+  [AddCommGroup F] [TopologicalSpace F] [TopologicalAddGroup F]
+  [Module 𝕜 E] [Module 𝕜 F] [Module 𝕜₁ E] [Module 𝕜₂ F]
+
+open Topology
+
+variable (σ E F) in
+/-- The space of continuous linear maps equipped with the topology of pointwise/simple convergence,
+sometimes also called the strong operator topology.
```

Comment:
```suggestion
sometimes also called the *strong operator topology*. We avoid this terminology
since so many other things share similar names, and using "pointwise convergence"
in the name is more informative.
```